src/data/firebase.ts

```diff
@@ -87,7 +114,7 @@ export async function saveToFirebase(
   } as FirebaseStoredScene;
 
   const db = firebase.firestore();
-  const docRef = db.collection("scenes").doc(roomId);
+  const docRef = db.collection("scenes").doc(roomID);
```

Comment:
I would keep`roomId` for consistency

Reply:
`roomID` is the consistent name actually.. I wanted to rename all the local vars in that file and forgot, will do it now!